pkg/obfuscator/mac_address.go

```diff
@@ -58,7 +52,10 @@ func NewMacAddressObfuscator(replacementType schema.ObfuscateReplacementType) (O
 
```

Comment:
same as the IP obfuscator: can't comment on the above line, but you can remove the if replacementType != ... checks now